gigascience/gigadb-website
# Fix footer positioning in all pages

# Pull request for issue: #1456

This is a pull request for the following functionalities:

* Fix footer positioning to the bottom of the viewport

## How to test?

Navigate pages with small amount of content and verify that footer is positioned at bottom of page
E.g.
http://gigadb.gigasciencejournal.com:9170/site/login
http://gigadb.gigasciencejournal.com:9170/site/advisory
http://gigadb.gigasciencejournal.com:9170/site/term
http://gigadb.gigasciencejournal.com:9170/site/admin

## How have functionalities been implemented?

Turned body into flexbox and adjusted height to be the same of the viewport, so content within can be better positioned

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
